lib/segment/src/index/sparse_index/sparse_vector_index.rs

```diff
@@ -158,13 +162,14 @@ impl<TInvertedIndex: InvertedIndex> VectorIndex for SparseVectorIndex<TInvertedI
         let borrowed_id_tracker = self.id_tracker.borrow();
         let deleted_bitslice = borrowed_vector_storage.deleted_vector_bitslice();
         let mut ram_index = InvertedIndexRam::empty();
-        let available_point_count = borrowed_id_tracker.available_point_count();
+        let mut index_point_count: usize = 0;
```

Comment:
switched to actually counting the points to avoid inconsistencies

Reply:
agreed, we count points that are not deleted in id tracker and vector storage both